services/cd-service/pkg/history/history_test.go

```diff
@@ -294,3 +297,88 @@ func assertChangedAtNthCommit(t *testing.T, actualCommit *git.Commit, expectedPo
 	}
 	t.Errorf("wrong changed commit, expected %d, actually not any known commit", expectedPosition)
 }
+
+func BenchmarkHistoryNew(b *testing.B){
+	benchmarkHistory(b, false)
+}
+
+func BenchmarkHistoryOld(b *testing.B){
+	benchmarkHistory(b, true)
+}
```

Comment:
these functions seem unused.

Reply:
These are the Benchmark methods. They are used when you run test with `-bench .`

Reply:
ahh, nice :)